app/src/test/java/com/futurice/freesound/feature/search/SearchViewModelTest.java

```diff
@@ -86,6 +88,24 @@ public void getSounds_emitsSearchResultsWrappedInDisplayableItems() {
         ts.assertValue(ofObj(expectedDisplayableItems(sounds)));
     }
 
+    @Test
+    public void search_recoversFromSearchErrors() {
+        ArrangeBuilder arrangeBuilder = new ArrangeBuilder().withErrorWhenSearching();
```

Comment:
It's (AAA)^2 :-) I'll if I can do this neatly without adding to much to the ArrangeBuilder.